docs/guides/deployment/gcp.rst

```diff
@@ -0,0 +1,203 @@
+
+   $ gcloud init
+
+Create a project
+================
+
+If you already have a project you can use your existing project.
+
+.. code-block:: bash
+
+   $ PROJECT=edgedb
+
+   $ gcloud projects create $PROJECT
+
+Choose a name
+=============
+
+We'll add suffixes to this name to create names for each component in the
+deployment.
+
+.. code-block:: bash
+
+   $ NAME=edgedb
+
+Provision a Postgres instance
+=============================
+
+Choose your own secure ``ADMIN_PASSWORD``. This will be the password for both
+popostgres and EdgeDB.
+
+.. code-block:: bash
+
+   $ read -rsp "Admin Password: " ADMIN_PASSWORD
+
+   $ gcloud services enable container.googleapis.com \
+       --project=$PROJECT
+
+   $ gcloud sql instances create ${NAME}-postgres \
+       --database-version=POSTGRES_12 \
+       --cpu=1 \
+       --memory=3840MiB \
+       --region=us-west2 \
+       --project=$PROJECT
+
+   $ gcloud sql users set-password postgres \
+       --instance=${NAME}-postgres \
+       --password=$ADMIN_PASSWORD \
+       --project=$PROJECT
+
+Create a Kubernetes cluster
+===========================
+
+Before creating the cluster be sure that the Kubernetes Engine API is enabled .
+
+.. code-block:: bash
+
+   $ gcloud services enable container.googleapis.com \
+       --project=$PROJECT
+
+Create a cluster.
+
+.. code-block:: bash
+
+   $ gcloud container clusters create ${NAME}-k8s \
+       --zone=us-west2-a \
+       --num-nodes=1 \
+       --project=$PROJECT
+
+Configure Cloud SQL proxy credentials
+=====================================
+
+.. code-block:: bash
+
+   $ gcloud services enable iam.googleapis.com \
+       --project=$PROJECT
+
+   $ gcloud iam service-accounts create ${NAME}-account \
+       --project=$PROJECT
+
+   $ gcloud services enable sqladmin.googleapis.com \
+       --project=$PROJECT
+
+   $ MEMBER="serviceAccount:${NAME}-account@${PROJECT}.iam.gserviceaccount.com"
+   $ gcloud projects add-iam-policy-binding $PROJECT \
+       --member=$MEMBER \
+       --role=roles/cloudsql.admin \
+       --project=$PROJECT
+
+   $ gcloud iam service-accounts keys create credentials.json \
+       --iam-account=${NAME}-account@${PROJECT}.iam.gserviceaccount.com
+
+   $ kubectl create secret generic cloudsql-instance-credentials \
+       --from-file=credentials.json=credentials.json
+
+   $ INSTANCE_CONNECTION_NAME=$(
+       gcloud sql instances describe ${NAME}-postgres \
+           --format="value(connectionName)" \
+           --project=$PROJECT
+   )
+
+   $ DSN="postgresql://postgres:${ADMIN_PASSWORD}@127.0.0.1:5432"
+   $ kubectl create secret generic cloudsql-db-credentials \
+       --from-literal=dsn=$DSN \
+       --from-literal=password=$ADMIN_PASSWORD \
+       --from-literal=instance=${INSTANCE_CONNECTION_NAME}=tcp:5432
+
+Deploy EdgeDB
+=============
+
+.. code-block:: bash
+
+   $ URL="https://raw.githubusercontent.com"
+   $ URL="${URL}/edgedb/edgedb-deploy/dev/gcp/deployment.yaml"
+   $ wget $URL
```

Comment:
Is there a better way to write this url so that it doesn't violate the 80 character line length limit? This looks strange in a guide:
![image](https://user-images.githubusercontent.com/8278635/151065954-d711049d-2a47-465e-80dd-9eeca3480035.png)

If I use a backslash the second line is treated as output rather than part of the command and the copy button doesn't copy it.
![image](https://user-images.githubusercontent.com/8278635/151066593-ea6208a5-fc3d-40c5-9620-b236872d92a1.png)

cc @elprans

Reply:
This is a site bug, backslash should be interpreted correctly.